tests/record_test.py

```diff
@@ -378,8 +377,8 @@ def test_transform_without_update_returns_same_precord():
 
 
 class Application(PRecord):
-    name = field(type=(six.text_type,) + six.string_types)
-    image = field(type=(six.text_type,) + six.string_types)
+    name = field(type=(str, str))
```

Comment:
Not quite sure I fully understand the relation between the two PRs that you've submitted. Some changes are present in both while others are only present in one of the branches. Would it be possible to add these two commits to one branch and make a PR of that with the complete changeset?

Reply:
Done, and i'll close this pr